src/main/webapp/usermap.html

```diff
@@ -55,9 +55,9 @@
             <h1 id="caption">Who is using TEAMMATES?</h1>
             <div id="contentHolder">
                 <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE"></div>
-                <p id="totalCount"></p>
-                <p>Last updated: 14 March 2016</p>
-            </div>
+                <p id="lastUpdate" class="lastUpdate">Last updated: 14 March 2016</p>
+                <h2 class="subcaptionTypeTwo"><span id="totalUserCount" class="submissionsNumber"></span> institutions from <span id="totalCountryCount" class="submissionsNumber"></span> countries</h2>
```

Comment:
Two issues here:
1. Use the `totalCount` that you defined instead of `submissionsNumber` here although both have the same style, to be more representative.
2. The line is too long. Consider breaking it, e.g:
```html
<h2>
    ...
</h2>
```